form/icon-button.ftd

```diff
@@ -279,14 +294,20 @@ $on-mouse-enter$: $ftd.set-bool($a = $button-wrapper.hover, v = true)
 $on-mouse-leave$: $ftd.set-bool($a = $button-wrapper.hover, v = false)
 spacing.fixed.px: 8
 width: hug-content
-min-width: auto
+min-width.fixed.px: 322
 link: $button-wrapper.link
-color: $ds.colors.text-strong
+color if { button-wrapper.kind == NULL }: $ds.colors.text-strong
```

Comment:
`button-wrapper.kind` is not optional. You can make it optional in component's header.